__tests__/locale_files.test.js

```diff
@@ -0,0 +1,15 @@
+jest.unmock('fs-extra');
+import fs from 'fs-extra';
+import path from 'path-extra';
+
+describe('Tests for locale files', () => {
+  test('Verify that only English and Hindi files exist', () => {
```

Comment:
@richmahn we'll have to update this test any time we add a new locale.
It would be better to test that an invalid locale file does not exist.
e.g. check that none of the locale files contain a `%` symbol in the file name.

Reply:
Fixed